app/javascript/components/Profile.js

```diff
@@ -0,0 +1,234 @@
+      }}
+    </Mutation>
+  );
+};
+const UpdateGithubOrg = props => {
+  let input;
+  return (
+    <Mutation mutation={UPDATE_GITHUB_ORG}>
+      {(updateGithubOrg, { data, error, loading, called }) => {
+        if (error) {
+          return <div>{JSON.stringify(error)}</div>;
+        }
+        if (loading) {
+          return <div>Submitting...</div>;
+        }
+        if (called) {
+          return <div className="text-center pt-3">Github Org Updated!</div>;
+        }
+        const { orgs } = props
+        return (
+          <div>
+            <form
+              className="w-full"
+              onSubmit={e => {
+                e.preventDefault();
+                updateGithubOrg({
+                  variables: { organizationId: parseInt(input.value, 10) }
+                });
+                input.value = "";
+              }}
+            >
+              <div className="flex items-center border-b border-b-2 border-teal py-2">
+                <select
+                  className="block appearance-none w-full bg-grey-lighter border border-grey-lighter text-grey-darker py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-grey"
+                  id="grid-state"
+                  ref={node => {
+                    input = node;
+                  }}
+                >
+                  {orgs.map(org => (
+                    <option value={org.id}>{org.name}</option>
+                  ))}
+                </select>
+                <div className="pointer-events-none absolute pin-y pin-r flex items-center px-2 text-grey-darker">
+                  <svg
+                    className="fill-current h-4 w-4"
+                    xmlns="http://www.w3.org/2000/svg"
+                    viewBox="0 0 20 20"
+                  >
+                    <path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
+                  </svg>
+                </div>
+                <button
+                  className="flex-no-shrink bg-teal hover:bg-teal-dark border-teal hover:border-teal-dark text-sm border-4 text-white py-1 px-2 rounded"
+                  type="submit"
+                >
+                  Update Github Organization
+                </button>
+              </div>
+            </form>
+          </div>
+        );
+      }}
+    </Mutation>
+  );
+};
+
+export default class Profile extends Component {
+  render() {
+    return (
+      <Query query={LOAD_USER_PROFILE}>
+        {({ loading, error, data }) => {
+          if (loading) return <p>Loading...</p>;
+          if (error) return <p>Error</p>;
+          return (
+            <div className="flex-col w-1/2 mt-10 p-5 bg-gray m-auto bg-grey-lightest border border-grey rounded">
+              <UpdateAccessToken hasAccessToken={data.me.hasAccessToken} />
+              <UpdateGithubUsername githubUsername={data.me.githubUsername} />
+              <UpdateGithubOrg
+                orgs={MOCK_ORGS}
```

Comment:
I pulled this branch down and all is working nicely!  Can we now remove these mocks, and replace it with an actual query for the database's actual `Organization` records' details here?

Reply:
Just to get this merged without falling too far behind, i opened #70 for that work